JetNews/app/src/main/java/com/example/jetnews/ui/interests/InterestsScreen.kt

```diff
@@ -159,10 +156,47 @@ fun InterestsScreen(
         }
     ) { innerPadding ->
         val screenModifier = Modifier.padding(innerPadding)
-        TabContent(currentSection, isExpandedScreen, onTabChange, tabContent, screenModifier)
+        InterestScreenContent(currentSection, isExpandedScreen, onTabChange, tabContent, screenModifier)
     }
 }
 
+@Composable
+fun rememberTabContent(interestsViewModel: InterestsViewModel): List<TabContent> {
```

Comment:
This could do with more documentation. Can you point to a DAC page that introduces composables that don't emit UI?

Reply:
Added! Actually, there's nothing in the docs we can point to. I couldn't find a nice place for where that would live either 🤔